x-pack/qa/multi-cluster-tests-with-security/build.gradle

```diff
@@ -30,6 +30,11 @@ testClusters {
 
   'mixed-cluster' {
     testDistribution = 'DEFAULT'
+    numberOfNodes = 2
+    // Node roles are configured this way in order to verify redirecting the transform request from the node lacking
+    // remote_cluster_client role to the node that is remote_cluster_client.
+    nodes."mixed-cluster-0".setting 'node.roles', '[data,ingest,master,transform]'
+    nodes."mixed-cluster-1".setting 'node.roles', '[data,ingest,master,transform,remote_cluster_client]'
```

Comment:
Similar comment to above about backporting complications.  Do _any_ tests in this suite use the node client rather than the REST client on master?  If so those tests will randomly use either the node or transport client on 7.x and when they choose the transport client there will be problems with specifying the `transform` role here in 7.x.